qiskit_experiments/calibration_management/base_calibration_experiment.py

```diff
@@ -44,54 +44,53 @@ class BaseCalibrationExperiment(BaseExperiment, ABC):
+    This ensures that the `run` method of `RoughFrequency` will be the
+    `run` method of the `BaseCalibrationExperiment` class. Furthermore, developers
+    must explicitly call the `__init__` methods of both parent classes.
 
     Developers should strive to follow the convention that the first two arguments of
-    a calibration experiment are the qubit(s) and the :class:`.Calibrations` instance.
+    a calibration experiment are the qubit(s) and the :class:`~qiskit_experiments.calibration_management.Calibrations` instance.
 
     If the experiment uses custom schedules, which is typically the case, then
-    developers may chose to use the :meth:`get_schedules` method when creating the
-    circuits for the experiment. If :meth:`get_schedules` is used then the developer
-    must override at least one of the following methods used by :meth:`get_schedules`
+    developers may choose to use the `get_schedules` method when creating the
+    circuits for the experiment. If `get_schedules` is used, then the developer
+    must override at least one of the following methods used by `get_schedules`
     to set the schedules:
 
-    #. :meth:`_get_schedules_from_options`
+    #. `_get_schedules_from_options`
+    #. `_get_schedules_from_calibrations`
+    #. `_get_schedules_from_defaults`
 
-    #. :meth:`_get_schedules_from_calibrations`
+    These methods are called by `get_schedules`.
 
-    #. :meth:`_get_schedules_from_defaults`
-
-    These methods are called by :meth:`get_schedules`.
-
-    The :meth:`update_calibrations` method is responsible for updating the values of the parameters
-    stored in the instance of :class:`.Calibrations`. Here, :class:`BaseCalibrationExperiment`
-    provides a default update methodology that subclasses can override if a more elaborate behaviour
-    is needed. At the minimum the developer must set the variable :code:`_updater` which
-    should have an :code:`update` method and can be chosen from the library
-    :mod:`qiskit_experiments.calibration_management.update_library`. See also
-    :class:`qiskit_experiments.calibration_management.update_library.BaseUpdater`. If no updater
-    is specified the experiment will still run but no update of the calibrations will be performed.
+    The `update_calibrations` method is responsible for updating the values of the parameters
+    stored in the instance of :class:`~qiskit_experiments.calibration_management.Calibrations`. Here, `BaseCalibrationExperiment`
+    provides a default update methodology that subclasses can override if a more elaborate behavior
+    is needed. At the minimum, the developer must set the variable `_updater` which
+    should have an `update` method and can be chosen from the library
+    :mod:`~qiskit_experiments.calibration_management.update_library`. See also
+    :class:`~qiskit_experiments.calibration_management.update_library.BaseUpdater`. If no updater
```

Comment:
This link won't work because `BaseUpdater` isn't currently exposed in the API toctree. You need to add `from .update_library import BaseUpdater` to `calibration_management/__init__.py`.